File: scripts/rotate_robot.py
#!/usr/bin/env python
import sys

import rospy
from geometry_msgs.msg import Twist

BURGER_MAX_ANG_VEL = 2.84

if __name__=='__main__':
	rospy.init_node('rotate_robot', anonymous=True)

	pub = rospy.Publisher('cmd_vel', Twist, queue_size=10)
	twist = Twist()
	twist.angular.x = 0.0; twist.angular.y = 0.0; 

	try:
		while not rospy.is_shutdown():
			twist.angular.z = BURGER_MAX_ANG_VEL
			pub.publish(twist)

	except:
		print("Unexpected error:", sys.exc_info()[0])

	finally:
	    twist.linear.x = 0.0; twist.linear.y = 0.0; twist.linear.z = 0.0
	    twist.angular.x = 0.0; twist.angular.y = 0.0; twist.angular.z = 0.0
	    pub.publish(twist)

File: team6_object_follower/src/rotate_robot.cpp
#include "ros/ros.h"
#include "geometry_msgs/Twist.h"

#define BURGER_MAX_ANG_VEL 2.84

int main(int argc, char **argv)
{
    ros::init(argc, argv, "rotate_robot", ros::init_options::AnonymousName);
    ros::NodeHandle n;

    ros::Publisher pub = n.advertise<geometry_msgs::Twist>("cmd_vel", 10);

    geometry_msgs::Twist twist;
    while (ros::ok())
    {
        twist.angular.z = BURGER_MAX_ANG_VEL;
		pub.publish(twist);
    }
}